### DISCUSSION TITLE
Getting 50% less money in stripe via github sponsors

### DISCUSSION THREAD

**Discussion by nischay876** [community/community#61595] [Other Feature Feedback, Questions, & Ideas]
so my customer paid me $75 and i only got 2000INR in stripe
but $75 is like 6000INR
before saying contact stripe read this https://github.com/orgs/community/discussions/61595#discussioncomment-6524643
![image](https://github.com/community/community/assets/84630660/811417e9-552f-4768-8fc9-48b62f0d3d0c)
![image](https://github.com/community/community/assets/84630660/bf18aba6-e216-4408-888e-dd484a6b7139)

**Comment by GhostOf0days** [comment]
This is not an issue with Github. Please contact Stripe customer support.

**Reply by nischay876** [reply]
it is with github as the 1st ss is of github and it says only $30 processed and i asked stripe they said to contact github 
like how is it settled at $30 when the original amount sended was $75 and i asked that person who sended me money he said that $75 has been deducted from his bank

github is not responding on my ticket since 3+ days so here i am on discussions
the payments are settled but they are 50% less from the original amount given by my sponsors

![image](https://github.com/community/community/assets/84630660/3398bad0-6747-4e19-9b41-6e70458ee48a)

**Reply by GhostOf0days** [reply]
Please [create a support ticket](https://support.github.com/) with Github Support and all details.

**Reply by nischay876** [reply]
already did 4 days ago their support dead or something i think
or maybe they only response to paid customers...

**Reply by GhostOf0days** [reply]
They usually respond in a day or two. Please try again. We can't help you on community posts.

**Reply by nischay876** [reply]
the reason of posting the here was that i wanted to know if anyone else has this issue or is there is something behind processed part like i want some answers for example 
ans 1 money is gone i will never get it
ans 2 i will get the money next month or soon maybe
ans 3 i should never use github sponsors again 
like i wanna know if thats something happened to anyone before or something wrong from my side or from github's side ad i really think from $75 30-40 cannot be gone as fee

**Comment by Binary-Ninja-007** [comment]
Good.
